docs/docs/upgrade/network-api-migration.md

```diff
@@ -0,0 +1,699 @@
+---
+title: Migrating to 1.5 Networking API
```

Comment:
duplicate document? I just review the one above and this looks like the same file?